### ISSUE TITLE
Complete the manual

### ISSUE THREAD

**Issue by fingolfin** [gap-packages/NormalizInterface#4]

**Comment by fingolfin** [comment]
Make sure to get rid of all TODOs ;-).

**Comment by fingolfin** [comment]
Document `NmzHasConeProperty`

**Comment by fingolfin** [comment]
You can get a list of all variables whose name starts with `Nmz` like this:
```
Filtered(IDENTS_BOUND_GVARS(), function(x) return Length(x)>3 and x{[1..3]}="Nmz"; end);
```
And a list of all which are not documented like this:
```
Filtered(IDENTS_BOUND_GVARS(), function(x) return Length(x)>3 and x{[1..3]}="Nmz" and not IsDocumentedWord(x); end);
```

**Comment by fingolfin** [comment]
I just pushed some more changes. There are still various "TODO" in the manual, though. Maybe the most glaring instance is with `NmzConeProperty`, as tons of other manual entries just say: 
```
This is an alias for NmzConeProperty( cone, "BLABLA" ); see NmzConeProperty (2.2-6)
```
I guess the `NmzConeProperty` doc entry should briefly say what it does, and give a list of supported property names, and perhaps a one-sentence description what each means, and otherwise a pointer to the Normaliz interface. Perhaps the description should also describe what type the result is (an int, and int vector, a matrix, ...)

**Comment by csoeger** [comment]
All methods have at least a documentation stub. I started to add more content to them. For `NmzConeProperty`I will give a list of possible parameters and reference to the corresponding NmzBLALBLA function for details about meaning and return type. I think that is the better place for it. We did not do it this way at the beginning because those were auto-generated, including doc.

**Comment by csoeger** [comment]
I think it is complete now.
It currently contains some `#! <List> </List>` as a little hack for creating new lines. Can this be done in a nicer way?

**Comment by fingolfin** [comment]
According to the [GAPDoc manual](http://www.math.rwth-aachen.de/~Frank.Luebeck/GAPDoc/doc/chap3.html#X7A0D26B180BEDE37) you should be able to use `<Br>`, `<Par>` or `<P>` -- please take a look at the manual for details.

**Comment by csoeger** [comment]
Replaced it.
Only open part in the documentation is the last chapter "4 Installing NormalizInterface".
Should we just add a hint to the README or duplicate it?

**Comment by fingolfin** [comment]
I applied various additional tweaks and fixes to the manual in 3e31d2742970e99e643275757620e98f84d2436b

**Comment by csoeger** [comment]
Went over it once more. Looks good to me now.
